docs/versioning.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Versioning and maintenance policy
+description: Versioning and maintenance policy for Powertools for AWS Lambda (Python)
+---
+
+<!-- markdownlint-disable MD041 MD043 MD013 -->
```

Comment:
Please reenable `MD013` before merging it.

Reply:
suggestion: keep it to prevent changing pre-approved legal wording.